content/docs/how-we-work/baseline/logging-details.md

```diff
@@ -0,0 +1,145 @@
+   - Select the ticketing system (in our case commonly - Jira) in the **Project Settings** > **Integrations** to integrate Sentry with your ticketing system. Follow the prompts to authenticate and authorize Sentry to access your Jira instance. It is vital to Configure the integration settings, such as default project and issue type, to streamline ticket creation. For more details, refer to the [Sentry Jira Integration Documentation](https://docs.sentry.io/product/integrations/issue-tracking/jira/).
+
+3. **Create a Ticket:**
+   - **Manually Creating a Ticket:**
+     - Create an Issue** in the alert details view to manually create a ticket by writing all relevant information provided by Sentry to ensure the issue is well-documented.
+
+   - **Automatically Creating Tickets:**
+     - Configure automatic ticket creation in **Project Settings** > **Alerts** by setting up alert rules to trigger ticket creation by specifying the conditions (e.g., error type, frequency) and actions (create a Jira ticket) for automatic ticket creation.
+
+4. **Managing Tickets in Jira:**
+   - Ensure tickets are assigned to the appropriate team or individual in Jira. Use Jira’s workflow management tools to track the progress of the ticket from creation to resolution.
+   - **Tracking and Reporting:** Use Jira’s reporting features to track the status of logged issues. Generate reports to identify trends, recurring issues, and areas for improvement.
+   - **Comments and Updates:** Utilize Jira’s commenting system to add updates, discuss potential solutions, and collaborate with team members directly within the ticket.
+
+## Best Practices
+
+### Ensure Comprehensive Logging
+
+1. **User Interactions:**
+   - Use Drupal's built-in logging functions such as `watchdog` to log significant user actions and interactions.
+   - Implement custom logging in custom modules using the `\Drupal::logger('custom_module')->info($message);` syntax to capture specific user events.
+
+2. **Database Queries:**
+   - Enable query logging in Drupal by configuring the database logging settings in `settings.php`.
+   - Use the Database Logging module (`dblog`) to log and review database queries within the Drupal admin interface.
+   - Implement custom logging for complex or critical database operations using hooks and custom logging functions.
+
+3. **API Calls:**
+   - Log all outgoing and incoming API requests and responses. Use the Guzzle middleware to log API requests made through Guzzle HTTP client.
+   - Implement logging in custom API integrations by capturing request and response data and logging it using `\Drupal::logger`.
+
+4. **Third-Party Integrations:**
+   - Ensure that all third-party services and integrations have appropriate logging in place. This can include logging successful and failed requests, responses, and any relevant data exchanges.
+   - Utilize existing logging features provided by third-party modules and extend them as needed to capture additional details.
```

Comment:
What do you mean by third-party integrations here? From what I understand, this is the same as the API calls above. If you are talking about using API to talk to third-party services, then maybe we just club these two.